guides/release/models/finding-records.md

```diff
@@ -47,28 +49,29 @@ the `[]` notation will not work--you'll have to use `objectAt(index)` instead.
 ### Querying for Multiple Records
 
 EmberData provides the ability to query for records that meet certain criteria.
-Calling [`store.query()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/query?anchor=query) will make a `GET` request with the passed object serialized as query params.
-This method returns a `PromiseArray` in the same way as `findAll`.
+Calling [`query()`](https://api.emberjs.com/ember-data/5.3/functions/@ember-data%2Fjson-api%2Frequest/query) will make a `GET` request with the passed object serialized as query params.
+This method returns a respone from the server in the same way as `findRecord`.
 
 For example, we could search for all `person` models who have the name of
 `Peter`:
 
 ```javascript
 // GET to /persons?filter[name]=Peter
-this.store.query('person', {
+import { query } from '@ember-data/json-api/request';
+
+const result = await store.request(query('person', {
   filter: {
     name: 'Peter'
   }
-}).then(function(peters) {
-  // Do something with `peters`
-});
+}));
+const person = result.content.data;
 ```
 
 ### Querying for A Single Record
 
-If you are using an adapter that supports server requests capable of returning a single model object,
-EmberData provides a convenience method [`store.queryRecord()`](https://api.emberjs.com/ember-data/release/classes/Store/methods/queryRecord?anchor=queryRecord) that will return a promise that resolves with that single record.
-The request is made via a method `queryRecord()` defined by the adapter.
+If you are using an builder that supports server requests capable of returning a single model object,
```

Comment:
This whole paragraph I think should be rewritten. Something like if you need to fetch single record you should use `findRecord` builder for your request. Maybe some native speaker can write it better